src/store/actions.ts

```diff
@@ -883,3 +639,433 @@ export const setStaffPayment: ActionIF = ({ commit }, staffPayment: StaffPayment
 export const setFolioNumber: ActionIF = ({ commit }, folioNumber: string): void => {
   commit('mutateFolioNumber', folioNumber)
 }
+/** Name Check actions
+ * TODO: move these into a factory if converting to composition api
```

Comment:
If you're not going to do this in this ticket/PR, I suggest you tag this with "FUTURE" instead 😁